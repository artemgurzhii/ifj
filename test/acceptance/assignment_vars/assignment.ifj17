Scope
Dim a as integer
a = -4500
Dim b as integer = -4500
End Scope
