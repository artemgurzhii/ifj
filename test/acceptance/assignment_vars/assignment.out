.IFJcode17
JUMP Scope
LABEL Scope
DEFVAR GF@a
MOVE GF@a int@-4500
DEFVAR GF@b
MOVE GF@b int@-4500
